src/animation/KeyframeTrack.js

```diff
@@ -26,61 +35,124 @@ THREE.KeyframeTrack.prototype = {
 
 	constructor: THREE.KeyframeTrack,
 
-	getAt: function( time ) {
+	TimeBufferType: Float64Array,
+	ValueBufferType: Float64Array,
```

Comment:
Float32 should be sufficient for all animation stuff.  Will make it even speedier and same memory.